scipy/sparse/linalg/_interface.py

```diff
@@ -394,6 +391,10 @@ def __call__(self, x):
     def __mul__(self, x):
         return self.dot(x)
 
+    def __truediv__(self, other):
+        if np.isscalar(other):
+            return _ScaledLinearOperator(self, 1.0/other)
```

Comment:
This will just return none if `other` isn't a scalar

Reply:
Thanks, and my bad for overlooking this. Fixed now.